### PR TITLE
Add Ash and Amy to list of blog authors

### PR DESCRIPTION
**Pull request by FloorD**
Adding Ash to our list of bloggers.

### CODE REVIEW

File: src/blogAuthors.ts [hunk 1]
```diff
@@ -153,6 +154,13 @@ export const AUTHORS: Record<string, Author> = {
 		image_url: 'https://github.com/floord.png',
 		email: 'noreply@tembo.io',
 	},
+	ash: {
+		name: 'Ash Kordar',
```
Comment (aishwaryaborkar) [inline src/blogAuthors.ts:158]: hi thank you for doing this! my last name is Borkar :)

File: src/blogAuthors.ts [hunk 2]
```diff
@@ -153,6 +154,13 @@ export const AUTHORS: Record<string, Author> = {
...
+		title: 'Head of Education',
```
Comment (aishwaryaborkar) [inline src/blogAuthors.ts:159]: I'm definitely not trying to take your role 😅 my title is Senior React Engineer

Comment (DarrenBaldwin07) [reply src/blogAuthors.ts]: lol Floor told me to merge this and I was like I don't think we have 2 Head of Education's. Can you commit any changes directly to this branch?

Comment (DarrenBaldwin07) [reply src/blogAuthors.ts]: Also, @FloorD I think I know why you do not have admin merge access -- I'll make a tweak to the codeowners that should fix this

Comment (FloorD) [reply src/blogAuthors.ts]: so sorry both... I had... a morning

**Pull request merged**